server/app/views/BaseHtmlLayout.java

```diff
@@ -93,7 +97,10 @@ public HtmlBundle getBundle(HtmlBundle bundle) {
     bundle.addStylesheets(viewUtils.makeLocalCssTag(TAILWIND_COMPILED_FILENAME));
 
     // Add Google analytics scripts.
-    bundle.addFooterScripts(getAnalyticsScripts(measurementId).toArray(new ScriptTag[0]));
+    measurementId.ifPresent(
```

Comment:
optional: you don't need () and {} for one liner lambdas.

id -> bundle.addFooterScripts(getAnalyticsScripts(id).toArray(new ScriptTag[0]))

or 

```
measurementId
  .map(id -> getAnalyticsScripts(id).toArray(new ScriptTag[0]))
  .ifPresent(bundle::addFooterScripts);
```

Reply:
Done.